src/python/CRABInterface/RESTTask.py

```diff
@@ -115,6 +115,16 @@ def webdir(self, **kwargs):
         yield row.user_webdir
 
 
+    def getpublishurl(self, **kwargs):
+        if 'workflow' not in kwargs or not kwargs['workflow']:
+            raise InvalidParameter("Task name not found in the input parameters")
+        workflow = kwargs['workflow']
+        try:
+            row = next(self.api.query(None, None, self.Task.GetPublishUrl_sql, taskname=workflow))
+        except StopIteration:
+            raise ExecutionError("Impossible to find task %s in the database." % kwargs["workflow"])
+        return row
```

Comment:
You could make this a `yield row` to return 
```
{"result": [
 ["https://cmsweb.cern.ch/dbs/prod/phys03/DBSWriter"]
]}
```
instead of 
```
{"desc": {"columns": ["tm_publish_dbs_url"]}, "result": [
 "https://cmsweb.cern.ch/dbs/prod/phys03/DBSWriter"
]}
```
The 'yield' one should be easier to parse